client/client.go

```diff
@@ -127,3 +127,30 @@ func (client *Client) SysInfo() (*SysInfo, error) {
 
 	return &sysInfo, nil
 }
+
+type Capability struct {
+	Name  string            `json:"name"`
+	Label string            `json:"label"`
+	Type  string            `json:"type"`
+	Attrs map[string]string `json:"attrs,omitempty"`
+}
+
+// GetCapabilities gets the list of capabilities from the REST API
+func (client *Client) GetCapabilities() ([]Capability, error) {
```

Comment:
s/Get//

Would also be nice to qualify in the documentation what are these capabilities being obtained:
```
// Capabilities returns the capabilities currently available for snaps to consume.
```
Note we don't need to talk about the REST API in the documentation of the client, much like we don't talk about the TCP connection under it.